sdk/cosmos/examples/pom.xml

```diff
@@ -0,0 +1,159 @@
+<!--
```

Comment:
For similar reasons as already stated this folder should be `sdk/cosmos/microsoft-azure-cosmos-examples`

Reply:
will cover it in follow up PR too.